streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -148,16 +149,16 @@ public TopologyTestDriverTest(final boolean eosEnabled) {
         }
     }
 
-    private final static class Record {
+    private final static class TTDTestRecord {
```

Comment:
This was a bit funny to run into by this point in the implementation. It turns out we already had a class called "Record", and now we need to reference both of them in this test. I felt like it was more readable to just give this class a new name instead of referencing it by fully qualified name.